crates/muvm/src/bin/muvm.rs

```diff
@@ -297,13 +298,14 @@ fn main() -> Result<()> {
             }
         }
         let hidpipe_path = Path::new(&run_path).join("hidpipe");
-        if hidpipe_path.exists() {
+        if spawn_hidpipe_server(hidpipe_path.clone()).is_ok() {
```

Comment:
Should this still be allowed to fail?

Reply:
Sounds reasonable to me.

Reply:
For consistency with `muvm-guest.rs` treatment of `muvm-hidpipe`, changed to fail if can't be spawned.